Kickstarter-iOS/AppDelegate.swift

```diff
@@ -169,6 +171,14 @@ internal final class AppDelegate: UIResponder, UIApplicationDelegate {
         self?.presentRemoteNotificationAlert($0)
       }
 
+    self.viewModel.outputs.configureFabric
```

Comment:
Could wrap lines 174 - 180 in the conditional that way nothing is subscribed to in `DEBUG`.